.fernignore

```diff
@@ -1,2 +1,3 @@
 # Specify files that shouldn't be modified by Fern
 README.md
+src/utils
```

Comment:
Out of curiosity why do we need to ignore this from fern?

Reply:
This will stop Fern from overwriting this file when publishing new versions (they have a runner the generates the code & pushes to this repo)